lib/core/include/irods/irods_default_paths.hpp.in

```diff
@@ -3,24 +3,34 @@
 
 #include <boost/filesystem/path.hpp>
 
-#define IRODS_DEFAULT_PATH_LIBDIR     "@CMAKE_INSTALL_LIBDIR@"
-#define IRODS_DEFAULT_PATH_SYSCONFDIR "@CMAKE_INSTALL_SYSCONFDIR@"
-#define IRODS_DEFAULT_PATH_HOMEDIR    "@IRODS_HOME_DIRECTORY@"
-#define IRODS_DEFAULT_PATH_PLUGINDIR  "@IRODS_PLUGINS_DIRECTORY@"
+#define IRODS_DEFAULT_PATH_SBINDIR       "@CMAKE_INSTALL_SBINDIR@"
+#define IRODS_DEFAULT_PATH_LIBDIR        "@CMAKE_INSTALL_LIBDIR@"
+#define IRODS_DEFAULT_PATH_SYSCONFDIR    "@CMAKE_INSTALL_SYSCONFDIR@"
+#define IRODS_DEFAULT_PATH_RUNSTATEDIR   "@CMAKE_INSTALL_RUNSTATEDIR@"
+#define IRODS_DEFAULT_PATH_HOMEDIR       "@IRODS_HOME_DIRECTORY@"
+#define IRODS_DEFAULT_PATH_PLUGINDIR     "@IRODS_PLUGINS_DIRECTORY@"
```

Comment:
@SwooshyCueb Notice the additions of `SBINDIR` and `RUNSTATEDIR.`

- `SBINDIR` was added so that we can launch the correct binaries regardless of install location
- `RUNSTATEDIR` was added so we can write the PID file to the correct directory regardless of install location

Thoughts?

Reply:
Seems fine to me.

Reply:
Cool cool. Resolving.